docs/openapi/components/schemas/common/HouseBillOfLadingCertificate.yml

```diff
@@ -0,0 +1,80 @@
+$linkedData:
+  term: HouseBillOfLadingCertificate
+  '@id': https://w3id.org/traceability#HouseBillOfLadingCertificate
+title: House Bill of Lading Certificate
+description: >- 
+  A bill of lading issued by a freight forwarder. Often covers a consignment of parcels from various shippers that has been grouped or consolidated by the forwarder. The forwarder may, for example, receive a single groupage bill of lading from the carrier, then issue a series of House B/Ls to the respective shipper.
+  (source: Olegario Llamazares: DICTIONARY OF INTERNATIONAL TRADE, Key definitions of 2000 trade terms and acronyms).
+type: object
+properties:
+  '@context':
+    type: array
+    const:
+      - 'https://www.w3.org/2018/credentials/v1'
+      - 'https://w3id.org/traceability/v1'
+  type:
+    type: array
+    const:
+      - VerifiableCredential
+      - HouseBillOfLadingCertificate
+  id:
+    type: string
+  name:
+    type: string
+  description:
+    type: string
+  issuanceDate:
+    type: string
+  issuer:
+    type: object
+  credentialSubject:
+    type: object
+  proof:
+    type: object
+  relatedLink:
+    title: Related Link
+    description: Links related to this verifiable credential
+    type: array
+    items:
+      $ref: ./LinkRole.yml
+additionalProperties: false
+required: []
+example: |-
+  {
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+      "https://w3id.org/traceability/v1"
+    ],
+    "id": "https://example.com/credential/123",
+    "type": [
+      "VerifiableCredential",
+      "HouseBillOfLadingCertificate"
+    ],
+    "name": "House Bill Of Lading",
+    "issuanceDate": "2022-03-04T13:40:00Z",
+    "issuer": {
+      "type": "Organization",
+      "id": "did:key:z6MktHQo3fRRohk44dsbE76CuiTpBmyMWq2VVjvV6aBSeE3U",
+      "name": "MULTI CONTAINER LINE",
+      "address": {
+        "type": "PostalAddress",
+        "organizationName": "MCL Multi Container Line LTD.",
+        "streetAddress": "Rm. 3501, 35/F Manhatten Place, 23 Wang Tai Road",
+        "addressLocality": "Kowloon Bay",
+        "addressRegion": "Hong Kong",
+        "addressCountry": "Hong Kong SAR"
+      }
+    },
+    "credentialSubject": {
+      "type": [
```

Comment:
Why is this a consignment not a bill of lading?
Both the master and the House Bill of lading appear identical in schema definition, unless I'm missing something.